src/kani-driver/src/args.rs

```diff
@@ -321,6 +324,14 @@ impl KaniArgs {
             )
             .exit();
         }
+
+        if self.harness_unwind.is_some() && self.harness.is_none() {
```

Comment:
Ah, I don't know if there's a good example but you should be able to put `requires` on the `structopt` flags. I think maybe there's an example of a `conflicts`. Then you don't need to write any code here.

(These checks are legacy of trying to avoid changing some tests when I was rewriting the python. If you're looking for another tiny cleanup a separate PR to remove these manual checks in favor of adding the correct annotations, then fixing the expected test, might be nice.)